src/components/PathwaysCard/PathwaysCard.jsx

```diff
@@ -0,0 +1,84 @@
+import { React, useState } from "react";
+
+import { CircularProgressbar, buildStyles } from "react-circular-progressbar";
+import "react-circular-progressbar/dist/styles.css";
+import "./PathwaysCard.scss";
+import image from "../../assets/images/financial-forest.svg";
+
+const PathwaysCard = ({ card, percentage }) => {
+    const { header, topics } = card;
+    const [overlayActive, setOverlayActive] = useState(false);
+
+    const handleClick = () => {
+        setOverlayActive(true);
+        console.log("Switched to overlay view");
+    };
+
+    const PathwayCardJSX = (
+        <>
+            <div className="card" data-testid="1" onClick={handleClick}>
+                    <div className="card__progress" data-testid="2">
+                        <CircularProgressbar
+                            className="card__progress-bar"
+                            value={percentage}
+                            text={percentage}
+                            strokeWidth={10}
+                            styles={buildStyles({
+                                textColor: "#3b5165",
+                                pathColor: "#b4ddc0",
+                                trailColor: "#e9e9e9",
+                                textSize: "22px",
+                            })}
+                        />
+                    </div>
+                    <div className="card__content">
+                        <img
+                            className="card__content-image"
+                            src={image}
```

Comment:
If we are passing in the image src as a prop we should make sure the alt tag is something dynamic as well. Perhaps the `header` prop?